modules/common/wikiquote.js

```diff
@@ -0,0 +1,161 @@
+var require_install = require('require-install'),
+  request = require_install('request');
+
+/**
+ * This is a *very* heavily modified version of the library found here:
+ * https://github.com/natetyler/wikiquotes-api
+ *
+ * Added by Jay Harris on 22/07/2015
+ */
+(function() {
```

Comment:
It was already there. I just left it in the function :P. I can take it out if you like.